pkg/appconsts/consensus_consts.go

```diff
@@ -4,5 +4,8 @@ import "time"
 
 const (
 	TimeoutPropose = time.Second * 10
-	TimeoutCommit  = time.Second * 10
+	// TargetHeightDuration is the intended block interval duration (i.e. the
+	// time between blocks). Note that this is a target because CometBFT does
+	// not guarantee a fixed block interval.
+	TargetHeightDuration = time.Second * 15
```

Comment:
@evan-forbes would appreciate your attention here

Reply:
Yup this looks good 👍